src/mvt/android/artifacts/tombstone_crashes.py

```diff
@@ -62,7 +62,7 @@ class TombstoneCrashResult(pydantic.BaseModel):
     process_name: Optional[str] = None
     binary_path: Optional[str] = None
     selinux_label: Optional[str] = None
-    uid: Optional[int] = None
+    uid: int
```

Comment:
note that "arch" above shouldn't be optional either. (github won't let me comment on a context line that's outside the usual context window.)

if you're seeing that, it might be a side effect of the fact that arm32 is encoded as 0 :-( (https://android.googlesource.com/platform/system/core/+/refs/heads/master/debuggerd/proto/tombstone.proto#85)